Tasks/AzureResourceGroupDeployment/task.json

```diff
@@ -240,6 +240,35 @@
             "defaultValue": true,
             "helpMarkDown": "Choose if the tags configured on the Azure VM need to be copied to the corresponding Deployment Group agent. <br><br>​By default all Azure tags will be copied following the format “Key: Value”. Example: An Azure Tag “Role : Web” would be copied  as-is to the Agent machine. <br><br>For more information on how tag Azure resources refer to [link](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-using-tags​)"
         },
+        {
```

Comment:
bump up the task version

Reply:
Specify the endpoint to connect to VSTS/TFS account for agent registration.  <br><br>You can create an endpoint using \"+ New\", and select "Token-based authentication". You need a [personal access token(](https://docs.microsoft.com/en-us/vsts/accounts/use-personal-access-tokens-to-authenticate?view=vsts)PAT) to setup an endpoint. The scope of the PAT can be restricted to \"Deployment Group\". <br><br>Click \"Manage\" to update endpoint details.​"